content/id/docs/concepts/workloads/pods/pod-preset.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Pod Preset
+content_template: templates/concept
+weight: 30
+---
+
+{{% capture overview %}}
+Halaman ini menyajikan gambaran umum tentang `Pod Preset`, yang merupakan objek untuk memasukkan informasi tertentu ke dalam `Pod` pada saat waktu penciptaan. Informasi dapat berupa _secret_, _volume_, _volume mount_, dan variabel _environment_.
```

Comment:
For API resource object (i.e. Pod) the convention is to use camelcase (with capital first letter) without backtick and without italic. But if the context of resource object is field name, you can use backtick.

ref: https://kubernetes.io/docs/contribute/style/style-guide/#use-camel-case-for-api-objects
https://kubernetes.io/docs/contribute/style/style-guide/#inline-code-formatting

same comment applies for the rest of the stuffs here, you can do self-judgement

Reply:
fixed in https://github.com/kubernetes/website/pull/14394/commits/bb4445855c8266473904a9566e5791344b91e11d

However, I left everything highlighted and space separated words from the original page.

There is one thing though, "Pod spec" and "Pod Spec" seems point to different meanings. For now, I assume that Pod spec is a pod specification in general, meanwhile Pod Spec is specific for `PodSpec` field of a Pod.